elyra/api/elyra.yaml

```diff
@@ -506,3 +570,10 @@ components:
       required: true
       schema:
         type: string
+    path:
+      name: path
+      in: path
+      description: The path to the file to be parsed, relative to the JupyterLab server root directory.
```

Comment:
Hmm.  I'm wondering if we should make this name slightly more specific (e.g., `filepath`) and the description, less specific (eg., _The path to the file relative to the JupyterLab server root directory._) since I would imagine we'll have other endpoints that take a file.

The other option would be to name this parameter specifically for this endpoint (e.g., `contents_properties_path`) and leave the description alone, but I think making it more generic might be best at this point.

Reply:
Sounds good. I'll change the parameter description to "The path to the file". Does this also apply to the description in line 353 or should we leave that one more specific?

Reply:
> Does this also apply to the description in line 353 or should we leave that one more specific?

Good question.  Currently, all parameter descriptions match their use in the endpoints section and their definition, but I would think the description in the endpoints section can specific to _that_ endpoint.  Let's leave the description alone in the endpoints sections and then see how the swager.io editor handles these - in terms of which description is shown and, if both, where those different values appear.  (More out of curiosity than anything else IMO).

Reply:
Yeah, I was curious about that too! In swagger.io, the description it uses is the one in its endpoint section. I can't see anywhere that the other description appears. I tried deleting the description altogether in the endpoint section thinking that it would default to the parameter description, but it doesn't. And in fact now that I'm testing it, deleting the whole parameters section has no affect at all as far as I can see. 

I think this is because we don't reference the parameters using `$ref` and instead are sort of re-defining them for each endpoint. In order to pull in the parameter description, we would have to have the following in the endpoint:

```
parameters:
    - $ref: '#/components/parameters/filepath'
```

Reply:
Ah, right.  As best as I can determine, the ability to use `$ref` for path parameters appears to be relatively new (OpenAPI 3.0).  Kinda seems like we should be using `$ref` for path parameters or remove their definitions from the bottom parameters section unless they also appear in responses. 

Hmm, since we may want richer descriptions per use, I'm leaning toward scrapping the bottom parameters section and not using references.  What do you think?

Reply:
Yeah I'm inclined to agree. Especially because we don't have so many endpoints that maintaining the descriptions per endpoint is too involved. I'll just go ahead and remove the parameters section in that case!